src/aui-form-builder/assets/aui-form-builder-settings-modal-core.css

```diff
@@ -38,14 +21,49 @@
     margin: 9px 0px 0px 5px;
 }
 
-.form-builder-field-settings-cancel {
-    border-width: 0;
-}
+@media only screen and (min-device-width : 320px) and (max-device-width : 736px) {
```

Comment:
It is about your code, `.form-builder-field-settings .modal-body` should be listed before ` .form-builder-field-settings .modal-content` in alphabetical order